deps.sh

```diff
@@ -6,5 +6,11 @@ set -e
 
 # Test dependencies:
 tarantoolctl rocks install luatest 0.5.5
```

Comment:
It's worth to update to 0.5.7.

Reply:
Why it is worth to update? (luatest doesn't have changelog, so I need to dig into commits to know about interesting changes).

Reply:
Added bump of luatest version in a separate commit.